src/utils/lineHeight.ts

```diff
@@ -0,0 +1,10 @@
+/** Convert line height percentage values to multipliers. */
+export function convertLineHeightPercentageToMultiplier(value: any): number {
+	if (typeof value === "number") {
+		return value / 100
+	} else if (typeof value === "string") {
```

Comment:
Why do you have the redundant check for string type if you're going to treat everything like a string anyway?